protocol/specification/descriptive.rst

```diff
@@ -127,43 +127,102 @@ Optional Module Properties
     Related issue: :issue:`008 Groups and Hierarchy`
 
 ``"meaning"``
-    A tuple, with the following two elements:
+   An unordered dictionary regarding the module meaning. It provides metadata that is useful for interpreting measurement data in a machine-readable format. It can have the keys ``function``, ``importance``, ``belongs_to``, ``link`` and ``key``, all of which are optional, with some restrictions. A meaning property can also be added on the :ref:`parameter-meaning <paremeter level>`.
```

Comment:
```suggestion
   A struct with data regarding the module meaning. It provides metadata that is useful for interpreting measurement data in an automatic fashion. It can have the keys ``function``, ``importance``, ``belongs_to``, ``link`` and ``key``, all of which are optional, with some restrictions. A meaning property can also be added on the :ref:`parameter-meaning <paremeter level>`.
```

Reply:
I guess with the 'struct' you want to refer to the SECoP datainfo. However, we never said that SECoP datainfo may be applied to properties. Therefore it might be more correct to speak about a JSON object here.

Reply:
Yes, Ill change it to JSON object, I just looked and on other properties dicts/structs are named JSON objects as well.